src/components/Header.jsx

```diff
@@ -1,13 +1,17 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Link } from "react-router-dom";
 import Auth from "./Auth";
+import UserContext from "../contexts/UserContext";
+import ThemeToggle from "./ThemeToggle";
 
 const Header = () => {
+  const user = useContext(UserContext);
```

Comment:
You created a use context here but it is not being used anywhere in the code so why do we unnecessarily create it?